server/server.go

```diff
@@ -338,3 +338,12 @@ func (s *Server) CurrentState() (map[string]interface{}, error) {
 			"used_memory_mb":  util.MemoryUsage()},
 	}, nil
 }
+
+func (s *Server) CurrentQueueState() (map[string]interface{}, error) {
+
+	queues := make(map[string]interface{})
+	s.Store().EachQueue(func(q storage.Queue) {
```

Comment:
I'd rather you push this logic down into the Handler and not create a public API for it here.

Reply:
In other words, move this code into statsQueueHandler.